domains/transfer/Transfer.API/API/Transfer/Api/Controllers/TransferAgreementsController.cs

```diff
@@ -80,15 +77,24 @@ public async Task<ActionResult> Create(CreateTransferAgreement request)
                 statusCode: 409);
         }
 
-        var receiverBearerToken = AuthenticationHeaderValue.Parse(httpContextAccessor.HttpContext?.Request.Headers["Authorization"]!);
-        var receiverWdeBase64String = await projectOriginWalletService.CreateWalletDepositEndpoint(receiverBearerToken);
+        var subject = user.Subject;
+        var cancellationToken = new CancellationToken();
```

Comment:
I dont think this will have any effect, use CancellationToken.None instead?

Reply:
Or add token as parameter

Reply:
It's used in wallet calls below

Reply:
But why new CancellationToken, It's never cancelled right?